scripts/sync.sh

```diff
@@ -10,11 +10,15 @@ source "${ROOT_DIR}/scripts/common.sh"
 
 SYNC_BRANCH_NAME="sync-$(printf '%(%Y-%m-%d)T\n' -1)"
 
+set +x
+export GITHUB_TOKEN=${GITHUB_TOKEN:=$(cat "$BOT_ACCESS_TOKEN")}
+set -x
```

Comment:
We should be able to ditch this now, right?

Reply:
Yep, good call.